frontend/src/questionTemplates/cop_disease.json-e

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
What are all of these json-e files?

Reply:
Not sure. They were in the old robokop repo, so I copied them over, but I never looked at them.

Reply:
I'm not seeing them in the old repo. Please remove them.